mstm/__init__.py

```diff
@@ -166,7 +176,7 @@ def __init__(self, jones_vec, stokes_vec, wavelength):
 #### testing ##
 ##############
 if __name__ == "__main__":
```

Comment:
put test in a separate file. Nosetests will detect any file with the word "test" in it and run the tests that it contains.